crates/motoko/tests/test_vm.rs

```diff
@@ -269,6 +269,17 @@ fn prim_close_value() {
     assert_("prim \"closeValue\" (#Function { env = {}; content = { input = (#Wild, { source_type = \"Known\"; span = { start = 5; end = 6 }; line = 1; col = 6 }); exp = (#Literal(#Unit), { source_type = \"Known\"; span = { start = 9; end = 11 }; line = 1; col = 10 }); sugar = true } })", "func _ = ()");
 }
 
+#[test]
+fn prim_open_core() {
+    assert_("(prim \"openCore\" ()).cont.cont_type", "\"Taken\"");
```

Comment:
As you may recall, the `Taken` variant was an artifact of overcoming linear rules for `Box` in the program AST.

So, now that we are looking at the `Core` in motion, it reminds me of our use of Box and not Rc in the program AST, as something that may affect our performance numbers for the canister profiling (not sure how much yet).